media/js/base/global.js

```diff
@@ -64,10 +64,15 @@ function init_android_download_links() {
 function init_lang_switcher() {
     $('#language').change(function(event) {
         event.preventDefault();
-        gaTrack(
-            ['_trackEvent', 'Language Switcher', 'change', $(this).val()],
-            function() {$('#lang_form').submit();}
-        );
+        //Google Analytics
+        window.dataLayer = window.dataLayer || [];
```

Comment:
So the reason for repeating this line is in the event that `core-dataLayer.html` doesn't load for some reason? If that's the reasoning, I'm not sure it's necessary. I think if that `include` didn't work, we'd have bigger problems on our hands.

@jgmize - Thoughts?